counter/api.py

```diff
@@ -54,6 +89,85 @@ def fetch_one(self, counter_id: int):
             Counter.objects.annotate_is_open(), pk=counter_id
         )
 
+    @route.post(
+        "{counter_id}/seller/add",
+        response={200: SellerIdsResponse, 404: SellerCounterOrUserNotFoundResponse},
+        permissions=[IsCounterAdmin],
+        summary="Add sellers to a counter",
+        description="Add one or more users as sellers to the specified counter.\n\n"
+        "Error codes:\n\n"
+        "- 404: Counter not found (code=1).\n\n"
+        "- 404: User(s) not found or not sellers (code=2).",
+    )
+    def add_sellers_to_counter(self, request, counter_id: int, data: SellerIdsSchema):
```

Comment:
```suggestion
    def add_sellers_to_counter(self, counter_id: int, user_ids: list[int]):
```
Le paramètre `request` n'est pas utilisé sur les routes d'API. Regarde juste comment c'est fait ailleurs. En plus, tu n'utilises même pas le paramètre.

Ne crée pas un nouveau schéma de données si c'est juste un wrapper autour d'un type simple.